refinery/core/models.py

```diff
@@ -1387,6 +1387,15 @@ def data_sets_query(self):
         )
         return quote(json.dumps({analysis_facet_name: self.uuid}))
 
+    def facet_name(self):
```

Comment:
Considering how the method is named, it might be worth being explicit and decorating this method as a `@property` ?

Reply:
Also should reuse `facet_name` [here](https://github.com/refinery-platform/refinery-platform/blob/develop/refinery/core/models.py#L1383) now that we have it